config/istio-config/fluent-bit-ingressgateway-config-map.yaml

```diff
@@ -0,0 +1,52 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: ingressgateway-fluent-bit-forwarder-config
+  namespace: istio-system
+data:
+  fluent-bit.conf: |-
+    [SERVICE]
+        Flush          1
+        Daemon         Off
+        Log_Level      info
+        Parsers_File   parsers.conf
+    [INPUT]
+        Name    tail
+        Tag     istio-ingressgateway
+        Path    /var/log/containers/istio-ingressgateway*istio-proxy*.log
+        Parser  json-test
+        Refresh_Interval 1
+    [FILTER]
+        Name parser
+        Match istio-ingressgateway
+        Parser json-test
+        Key_Name log
+    [FILTER]
+        Name lua
+        Match istio-ingressgateway
+        script transform_record.lua
+        call transform_record
+    [OUTPUT]
+        Name stdout
+        Match istio-ingressgateway
```

Comment:
The ingressgateway is configured to send its logs to stdout via istio config. Does this do something different than sending ingressgateway logs to stdout on the pod?

Reply:
oh, that's a good point, I didn't notice. if istio ingressgateway is already writing the proxy entries to standard out is there a reason this output is adding them to the fluent sidecars standard out too? was this debugging?

Reply:
Yeah, this was debugging. 😬